### DISCUSSION TITLE
What is the documentation workflow

### DISCUSSION THREAD

**Discussion by fretchen** [openkfw/TruBudget#1306] [Q&A] [answered]
We are testing the generation of [documentation from markdown files ](https://github.com/openkfw/d4dtools)and keep wondering about appropiate frameworks. We currently consider:

- [jekyll](https://jekyllrb.com/), however first tests are not super convincing.
- [docusaurus](https://docusaurus.io/) someone mentionned that you guys might be using it.
- [jupyterbook](https://jupyterbook.org/en/stable/intro.html) Kind of works nicely for documentation...

Do you have any suggestions there ?

**Comment by georgimld** [comment, marked as answer]
We are using docusaurus and we are quite happy with it. You can check our [trubudget-website repo](https://github.com/openkfw/trubudget-website) to see how we implemented it. Basically the documentation that we publish in the Trubudget [`docs`folder](https://github.com/openkfw/TruBudget/tree/main/docs) is automatically published on our [website](https://trubudget.net/docs/README) using docusaurus when we publish a release. Hope this helps and the answer is not too late!

**Reply by fretchen** [reply]
Thanks. We had actually started to use [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) in the meantime in two repos. It works quite nicely in the [open-geodata-model](https://openkfw.github.io/open-geodata-model/) and the [d4dtools ](https://github.com/openkfw/d4dtools).
